lib/chef/azure/commands/enable.rb

```diff
@@ -29,21 +29,19 @@ def initialize(extension_root, *enable_args)
   def run
     load_env
 
-    report_heart_beat_to_azure(AzureHeartBeat::NOTREADY, 0, "Enabling chef-service...")
-
     enable_chef
 
     if @exit_code == 0
       if @chef_client_error
-        report_heart_beat_to_azure(AzureHeartBeat::READY, 0, "chef-service is enabled. Chef client run failed with error- #{@chef_client_error}")
+        report_heart_beat_to_azure(AzureHeartBeat::READY, 0, "chef-#{@daemon} is enabled. Chef client run failed with error- #{@chef_client_error}")

```

Comment:
What if `"daemon": "none"` was applied here?

Reply:
Thanks I need to check this scenario.